redis/redis.go

```diff
@@ -151,5 +149,8 @@ func (store *RedisFeatureStore) Upsert(key string, f ld.Feature) error {
 func (store *RedisFeatureStore) Initialized() bool {
```

Comment:
This lets us use the redis store even if the streaming api is down and can never initialize.